src/main/resources/assets/flamingo_ooo/lang/de_de.json

```
@@ -8,16 +8,16 @@
 
   "entity.flamingo_ooo.flamingo": "Flamingo",
 
-  "item.flamingo_ooo.flamingo_spawn_egg": "Flamingo-Ei",
-  "item.flamingo_ooo.flamingo_egg": "Flamingoei",
+  "item.flamingo_ooo.flamingo_spawn_egg": "Flamingo-Spawn-Ei",
+  "item.flamingo_ooo.flamingo_egg": "Flamingo-Ei",
```

`minecraft:turtle_egg` ist Schildkrötenei auf Deutsch, Flamingoei ist also konsistenter

```suggestion
  "item.flamingo_ooo.flamingo_egg": "Flamingoei",
```